digits/dataset/images/classification/test_views.py

```diff
@@ -260,6 +260,21 @@ def check_textfiles(self, absolute_path=True, local_path=True):
         job_id = self.create_dataset(**data)
         assert self.dataset_wait_completion(job_id) == 'Done', 'create failed'
 
+class TestCreateDb(BaseViewsTestWithImageset):
+    def test_log_file(self):
+        job_id = self.create_dataset()
+        assert self.dataset_wait_completion(job_id) == 'Done', 'create failed'
+        info = self.dataset_info(job_id)
+        assert len(info['CreateDbTasks']) != 0
+        for task in info['CreateDbTasks']:
+            db_name = task['db_name']
+            assert db_name  and db_name != ''
+            path = task['create_db_log_file']
+            assert path and path != ''
+            assert os.path.isfile(path), "log file %s does not exist" % path
+        assert self.delete_dataset(job_id) == 200, 'delete failed'
+        assert not self.dataset_exists(job_id), 'dataset exists after delete'
```

Comment:
No need to explicitly delete the dataset. It will be deleted in `tearDownClass()`:
https://github.com/NVIDIA/DIGITS/blob/f3c4078d335fd8c2b36b829d0cf0258fc314e561/digits/dataset/images/classification/test_views.py#L82-L84